src/journal/journald-native.c

```diff
@@ -487,17 +487,21 @@ int manager_open_native_socket(Manager *m, const char *native_socket) {
 
         r = setsockopt_int(m->native_fd, SOL_SOCKET, SO_PASSCRED, true);
         if (r < 0)
-                return log_error_errno(r, "SO_PASSCRED failed: %m");
+                return log_error_errno(r, "Failed to enable SO_PASSCRED on journal native socket: %m");
+
+        r = setsockopt_int(m->native_fd, SOL_SOCKET, SO_PASSRIGHTS, false);
+        if (r < 0 && r != -ENOPROTOOPT)
+                log_debug_errno(r, "Failed to turn off SO_PASSRIGHTS on journal native socket, ignoring: %m");
```

Comment:
the native logging socket actually *does* use SCM_RIGHTS. You can send a memfd to journald.